server/app/app.py

```diff
@@ -100,16 +99,16 @@ def create_cli():
         help="bind to all interfaces (this makes the server accessible beyond this computer)",
         action="store_true")
     launch_group.add_argument("--port", help="port to run server on", type=int, default=5005)
-    launch_group.add_argument("--debug", action="store_true",
+    launch_group.add_argument("-v", "--verbose", action="store_true",
                               help="more verbose output, including outputting warnings and every REST request")
-    launch_group.add_argument("--flask-debug", action="store_true", help=argparse.SUPPRESS)
+    launch_group.add_argument("--debug", action="store_true", help=argparse.SUPPRESS)
     launch_group.add_argument("--no-open", help="do not launch the webbrowser", action="store_false",
                               dest="open_browser")
     launch_group.add_argument(
-        "--category-selection-limit",
+        "--max-category-items",
         type=whole_number,
         help="maximum number of categories to display on the front-end. "
```

Comment:
this is not accurate - it does not control the number of categories.

Reply:
Should I use the verbiage from the doc? 
> specifies a numeric limit for the cardinality of a categorical annotation, beyond which the annotation will not be available for user selection in the front-end